src/test/java/io/cdap/plugin/cloudai/p2p/FileCopyToGCSActionTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.cloudai.p2p;
+
+import io.cdap.cdap.etl.api.validation.CauseAttributes;
+import io.cdap.cdap.etl.api.validation.ValidationFailure;
+import io.cdap.cdap.etl.mock.validation.MockFailureCollector;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link FileCopyToGCSAction}.
+ */
+public class FileCopyToGCSActionTest {
```

Comment:
These are mostly performing config validations. Please move them in their own class and add new tests methods for testing FileCopyToGCSAction behavior.

Reply:
Added new tests for actions